### PR TITLE
Added MP configuration for slideshow

### PR DESCRIPTION
**Pull request by LuisReyes98**
This is has the configuration for the new way to add fadeshow with machu pichu

### CODE REVIEW

File: manifest.scl [hunk 1]
```diff
@@ -3,7 +3,7 @@ imports = {
   pitch-deck = "themes/pitch-deck",
   fps-pitch = "themes/fps-pitch",
   slideshow-template-theme = "themes/slideshow-template-theme",
-
+  fadeshow = "themes/slideshow-template-theme/stylesheets/fadeshow",
```
Comment (LuisReyes98) [inline manifest.scl:6]: Done this way so the slideshow can access to it

**Comment by naturallymitchell** [comment]
if the fadeshow dependency was part of the theme, then other apps would get it automatically as well

I think as long as there's a `manifest.scl` file in any depedency it downloads, it'll resolve it

**Comment by LuisReyes98** [comment]
not necesary any more do to new approach

**Comment by naturallymitchell** [comment]
nice

**Pull request closed without merging**